lib/runtime/application.ts

```diff
@@ -195,12 +198,24 @@ export default class Application extends Addon {
    * config/environment.js
    */
   private async createServer(port: number): Promise<void> {
-    await new Promise((resolve) => {
+      await new Promise((resolve) => {
       let handler = this.router.handle.bind(this.router);
       if (this.config.server.ssl) {
-        https.createServer(this.config.server.ssl, handler).listen(port, resolve);
+        let server = https.createServer(this.config.server.ssl, handler).listen(port, resolve);
```

Comment:
I think this code could be dried up a bit because we're pushing the exact same promise to the `drainers` array. The final result would look something like:

```js
let server;
if (this.config.server.ssl) {
    server = https.createServer(this.config.server.ssl) //...
} else {
    server = http.createServer() // ...
}
this.drainers.push(async function drainServer()) // ...
```